### ISSUE TITLE
juliawebstack.org broken

### ISSUE THREAD

**Issue by kmsquire** [JuliaLang/juliawebstack.org#12]
Right now, I get:

Site Disabled

Oh no! It looks like this Pages site has been disabled due to a misconfigured custom domain.

**Comment by aviks** [comment]
Maybe due to: https://github.com/blog/1925-github-pages-legacy-ip-brownout

cc: @zachallaun @danielmendel @astrieanna

**Comment by aviks** [comment]
Yes, I can see that the `juliawebstack.org` domain has an A record of `204.232.175.78` which is deprecated. This needs to be set to an A record of 
    `192.30.252.153` and 
    `192.30.252.154`

More info here: https://github.com/blog/1917-github-pages-legacy-ip-deprecation
It looks like @danielmendel  owns the domain.

**Comment by kmsquire** [comment]
Cc: @IainNZ @WestleyArgentum (of JuliaWeb)

**Comment by IainNZ** [comment]
I can't do anything about this

**Comment by WestleyArgentum** [comment]
Alas, neither can I... Short of getting another domain or waiting until this one comes up again...

**Comment by zachallaun** [comment]
Hopefully Daniel can pop in and fix this, though I know that he is quite busy. Perhaps the domain can be transferred to someone who has more time to support this site?

**Comment by despeset** [comment]
This is fixed - happy to transfer the domain registration and or DNS control to a more active contributor, leaving this issue open for now.
